vllmd-hypervisor/generate-runtime-image.sh

```diff
@@ -0,0 +1,593 @@
+#            -vv \
+
+#            --kernel /home/sdake/repos/arnold-from-computelify/vllmd/vllmd-hypervisor/CLOUDHV.fd \
+#            --kernel /var/lib/artificial_wisdom/hypervisor-fw \
+#                   path="${OUTPUT_PATH}",readonly=off,id=root \
+#            --cmdline "console=ttyS0 console=hvc0 auto=true priority=critical root=/dev/vda1 fw" \
+#    local kernel_command_line="console=ttyS0 auto=true priority=critical modules=disk-detect auto-mount=true url=file:///hd-media/preseed.cfg debconf/frontend=texthyyp debian-installer/framebuffer=false DEBIAN_FRONTEND=text break=top"
+#            --cmdline "DEBIAN_FRONTEND=text auto=true priority=critical" \
+#            root=/dev/vda1 fw" \
+#            --cmdline "console=tty0 ``kDEBIAN_FRONTEND=text auto=true priority=critical" \
+#            --disk path="${iso_path}",readonly=on,id=cdrom \
+#                   path="${preseed_disk}",readonly=on,id=perseed \
+#                   path="${OUTPUT_PATH}",id=root \
+#            rd.module_blacklist=nouveau,nvidiafb console=tty0 DEBIAN_FRONTEND=text auto=true priority=critical interface=auto url=file:///preseed.cfg" \
+#            --cmdline "install priority=critical auto=true fb=false url=http://www.xsuckin.com/prseed" \
+#            --cmdline "auto url=autoserver" \
+#            --cmdline "auto=true priority=critical" \
+            #--cmdline "console=ttyS0 auto=true priority=critical interface=auto url=file:///preseed.cfg" \
+#            --kernel "${HOME}/CLOUDHV.fd" \
+#                   path="${preseed_disk}" \
+#                   path="${OUTPUT_PATH}" \
+#            --console pty \
+#            --serial tty
+#            --kernel "hypervisor-fw" \
+
+        # early kernel messages, add console=ttyS0 to cmdline and set flags `--serial tty` and `--console off`.
+        #
+#            --cmdline "console=ttyS0 install nomodeset priority=critical auto=true preseed/file=/cdrom/preseed.cfg debian-installer/framebuffer=false DEBIAN_FRONTEND=text"
+#            --cmdline "console=ttyS0,115200 priority=critical auto=true preseed/file=/preseed.cfg debconf/priority=critical DEBCONF_DEBUG=5 debian-installer/framebuffer=false DEBIAN_FRONTEND=text nogui fb=false auto-install/enable=true preseed/interactive=false debconf/frontend=noninteractive DEBCONF_DEBUG=developer noprompt nomodeset" \
+#
+#
+# serial --unit=0 --speed=115200
+# terminal_input serial; terminal_output serial
+#
+
+# Extract kernel and initrd from the Debian netinst ISO if needed
+# (You can mount the ISO and copy these files out)
+#        debian-installer/locale=en_US keymap=us hostname=myserver domain=local netcfg/choose_interface=auto url=http://your-server/preseed.cfg" \
+# --kernel "${HOME}/CLOUDHV.fd" \
+
+#                   path=t"${OUTPUT_PATH}",id=3 \
+#                   
+#
+
+#dd if=/dev/zero of=${OUTPUT_PATH} bs=4096 count=2048
+eza -l ${OUTPUT_PATH}
+        cloud-hypervisor-v44 \
+            --kernel $HOME/vmlinuz \
```

Comment:
add a comment with TODO(SECURITY) these files should be retrieved from the iso, not hard coded form who knows hwere.